AWS/legos/aws_check_ssl_certificate_expiry/aws_check_ssl_certificate_expiry.py

```diff
@@ -0,0 +1,56 @@
+##
+# Copyright (c) 2021 unSkript, Inc
+# All rights reserved.
+##
+import dateutil
+from pydantic import BaseModel, Field
+from typing import Dict,List
+import pprint
+import datetime
+
+
+class InputSchema(BaseModel):
+    aws_certificate_arn: str = Field(
+        title="Certificate ARN",
+        description="ARN of the Certificate"
+    )
+    region: str = Field(
+        title='Region',
+        description='Name of the AWS Region'
+    )
+
+
+def aws_check_ssl_certificate_expiry_printer(output):
+    if output is None:
+        return
+    pprint.pprint("Your SSL certificate is expiring in " + str(output) + " " + "days")
+
+
+def aws_check_ssl_certificate_expiry(
+    handle,
+    aws_certificate_arn: str,
+    region: str,
+) -> int:
+    """aws_check_ssl_certificate_expiry checks the expiry date of an ACM SSL certificate .
+
+            :type handle: object
+            :param handle: Object returned from Task Validate
+
+            :type aws_certificate_arn: str
+            :param aws_certificate_arn: ARN of the certificate
+
+            :type region: str
+            :param region: Region name of the AWS account
+
+            :rtype: Result Dictionary of result
+    """
+    iamClient = handle.client('acm', region_name=region)
+    result = iamClient.describe_certificate(CertificateArn=aws_certificate_arn)
+    for key,value in result['Certificate'].items():
+        if key == "NotAfter":
+            expiry_date = value
+            right_now = datetime.datetime.now(dateutil.tz.tzlocal())
+            diff = expiry_date-right_now
+            days_remaining = diff.days
+            if days_remaining < 30:
+                return days_remaining
```

Comment:
What happens if the days_remaining is more than 30 days, shouldn't we return the number of days.

Reply:
Added cases for days > 0, 30 > days < 0, and days < 0.

Days <30:
<img width="963" alt="1" src="https://user-images.githubusercontent.com/110628398/190947041-ff718d8b-c56f-45bf-90db-1f4f0a10648d.png">


Days <0 :
<img width="972" alt="Screenshot 2022-09-17 at 7 30 14 PM" src="https://user-images.githubusercontent.com/110628398/190946917-b09870a5-1591-4c0e-aa98-f60ec7905a41.png">

Days >0 (Default):
<img width="963" alt="Screenshot 2022-09-17 at 7 30 52 PM" src="https://user-images.githubusercontent.com/110628398/190946936-4e99b39f-7659-46d9-893b-6f55b79d7d81.png">